.chloggen/k8s-label-recommendations.yaml

```
@@ -0,0 +1,9 @@
+change_type: enhancement
+
+component: k8s
+
+note: How to populate resource attributes based on attributes, labels and transformation
+
+issues: [236]
+
+subtext:
```

not sure why CI is complaining about untidy yaml, maybe it want a newline here?
```suggestion
subtext:

```